azerothcore/azerothcore-wotlk
# feat(DB/locales) Import deDE quest locales from TrinityCore

## CHANGES PROPOSED:
Import the following commits from TrinityCore:

- https://github.com/TrinityCore/TrinityCore/commit/bf11c4acb593553c482eb82917fe33706022e9ef
- https://github.com/TrinityCore/TrinityCore/commit/7e1ff21cf81877171063403eac6e6eabd9e3e714
- https://github.com/TrinityCore/TrinityCore/commit/5f001ddb2ba7974696f34c8da32ae901faf81bf9

After importing the commits:
- Change `VerifiedBuild` from 0 to 18019 (TrinityCore uses 0 idk why)

Affects the following DB tables:

- `quest_offer_reward_locale`
- `quest_request_items_locale`


## ISSUES ADDRESSED:
- None



## TESTS PERFORMED:
- Imported SQL
- Checked a few texts ingame



## HOW TO TEST THE CHANGES:


- To test the changes you need a deDE Client
- Import SQL 
- Check a few of the imported texts ingame

## KNOWN ISSUES AND TODO LIST:

- we have 935 entries from `quest_request_items_locale` which already exists (Not really an issue because the texts are the same)


## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
